hack/policy/smoketest.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: EnterpriseContractPolicy
+metadata:
+  name: smoketest
+spec:
+  description: 'An example policy to test that the rule data here is valid'
+  publicKey: 'hack/policy/cosign.pub'
```

Comment:
This needs to be an absolute path.

Reply:
I don't think that did it. See 638ce91264d39dbd64b9da2b5a9b04e0e1dbda68.